modules/system-management/pages/system-metrics.adoc

```diff
@@ -0,0 +1,102 @@
+= Metrics Reporting
+:description: Overview of metrics gathering in TigerGraph Server.
+
+You can use `gadmin metric` commands to collect the following metrics related to state of resource usage on your instances:
+
+* CPU usage
+* Memory usage
+* Diskspace usage
+* Network-related information
+
+For a full list of options available to `gadmin metric`, run `gadmin metric -h` in the command line to view the help text for the command.
+
+== Report CPU usage
+
+Use command `gadmin metric -t cpu` to collect CPU usage by different TigerGraph services on a cluster or a node in the cluster.
```

Comment:
Does it just write to console? "collect" sounds to me like it creates a log, how about "report"?